src/components/controller.cpp

```diff
@@ -593,6 +593,10 @@ size_t controller::setup_modules(alignment align) {
     try {
       auto type = m_conf.get("module/" + module_name, "type");
 
+      if (type == tray_module::TYPE) {
+        m_tray_module_name = module_name;
+      }
```

Comment:
If we find a tray module and the tray module name is already set, we should print an error telling the user there can only be one tray module

Reply:
Doesn't really matter which tray module is then picked in the end, whatever is easier (but do tell the user in the error message).

Also, not starting superfluous modules would be a good idea as well.